README.md

```diff
@@ -82,13 +82,14 @@ Then visit `http://localhost:4200/`
 
 This is done using `CumcumberJS`, with the `Given`, `When`, `Then` format in English, in the `api/tests/features` directory.
 
-Internet is needed when running the automated API tests, because a real `jwt` is used from `Auth0` service.
+**Note**
+
+- Internet is needed when running the automated API tests, because a real `jwt` is used from `Auth0` service.
+- The API must NOT be running
```

Comment:
could consider running API on a different port to allow parallel development and testing

Reply:
Good idea, probably for a future improvement 🤓